eng/versioning/version_client.txt

```diff
@@ -63,6 +63,10 @@ com.microsoft.azure:azure-data-gremlin-spring-boot-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.2;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.2;2.3.3-beta.1
+com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.2.7;1.2.8-beta.1
```

Comment:
@chenrujun all of these libraries being added should not have dependency versions that were released from other repositories. If anything needs those versions they should be external dependencies. Since these are new libraries for this repository, the current and dependency versions should be the same until we've done the release from this repository.

Reply:
Thank you for your detailed explanation. I updated it.